roles/salto/templates/logback.xml.j2

```diff
@@ -11,13 +11,6 @@
         </encoder>
     </appender>
 
-    <logger name="io.sip3" level="INFO" additivity="false">
-        <appender-ref ref="DEFAULT"/>
-        {% if "webhook" in features -%}
-        <appender-ref ref="WEBHOOK"/>
-        {% endif -%}
-    </logger>
-
     {% if "webhook" in features -%}
     <appender name="WEBHOOK" class="io.sip3.logback.WebhookAppender">
```

Comment:
Starting from version 1.0.2 `logback-webhook-appender` configuration for appender was changed:
```xml
    <appender name="WEBHOOK" class="io.sip3.logback.WebhookAppender">
        <url>https://hooks.slack.com/services/....</url>
        <json>
            {
                "text": ":fire: Salto :fire:",
                "attachments": [
                    {
                        "text": "{message}"
                    }
                ]
            }
        </json>
        <interval>60</interval>
        <encoder>
            <pattern>%d{dd-MM-yyyy HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
        </encoder>
    </appender>
```

Filter section was removed and added property `interval` in seconds to set interval between messages.

Reply:
`<logback-webhook.version>1.0.1</logback-webhook.version>` in 2020.2.1 release...
Has this release `interval`?

Reply:
Yeap, sorry. We change it since 1.0.1 version not 1.0.2